include/linux/sched.h

```diff
@@ -1315,6 +1316,17 @@ struct task_struct {
 	struct callback_head		mce_kill_me;
 #endif
 
+ //needed in double fetch detection buffer to store address ranges 
+#ifdef CONFIG_DF_DETECTION
+        struct df_address_range *addresses;
```

Comment:
Please fix indentation of the struct fields (only use tabs to indent struct fields in the kernel)